src/object-properties.js

```diff
@@ -159,7 +159,7 @@ const properties = {
  * @property {boolean=} [style.bold = true] - Bold text
  * @property {boolean=} [style.italic = false] - Italic text
  * @property {boolean=} [style.underline = false] - Underlined text
- * @property {('responsive'|'fluid'|'fixed')} [sizeBehavior='responsive'] - Setting to determine how the fontsize of the label is calculated.
+ * @property {('responsive'|'relative'|'fixed')} [sizeBehavior='responsive'] - Setting to determine how the fontsize of the label is calculated.
```

Comment:
Has there been a release where we use `fluid`? In that case this is a breaking change. How come we want to rename this? It still says fluid in the property panel right?

Reply:
No we actually reverted it because of the issue I solved in this PR. As in kpi `fluid` has been set as `relative` Niek suggested to use the same here.

Reply:
ok good, then should we change the translation as well?

Reply:
`fluid` is what has been decided for it's translation, it's the same for kpi. It seems that users are supposed to see the option as 'fluid,' but in the code, it is called 'relative.' I'm agree it would be better to choose only one and keep using that everywhere but maybe in another PR after getting @niekvanstaveren's idea as well.